jobs/user-projects/general/nova-timetable.hcl

```diff
@@ -99,8 +126,13 @@ job "nova-timetable" {
 
       template {
         data        = <<EOH
-BOT_TOKEN={{ key "user-projects/nova/timetablebot/token" }}
-REDIS_ADDRESS={{ env "NOMAD_ADDR_db" }}
+BOT_TOKEN={{ key "user-projects/nova/bot/token" }}
+REDIS_ADDRESS={{ env "NOMAD_ADDR_redis" }}
+POSTGRES_USER={{ "user-projects/nova/postgres/user" }}
```

Comment:
This is missing the "key" value (line 131)

Reply:
Now how did that happen...